src/index.html

```diff
@@ -7,8 +8,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header_moyo">
+      <a href="#" class="nav_link_logo">
+        <img src="./images/logo.png" alt="logo"
+        class="img_logo">
+      </a>
+      <nav class="navigation">
+
+        <ul class="nav_ul">
+          <li class="nav_list">
+            <a href="#" class="nav_link is-active" >apple</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">samsung</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">smartphones</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link"
+            data-qa="hover" >laptops & computers</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">gadgets</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">tablets</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">photo</a>
+          </li>
+          <li class="nav_list">
+            <a href="#" class="nav_link">video</a>
+          </li>
+        </ul>
+
+      </nav>
```

Comment:
```suggestion
        </ul>
      </nav>
```
blank line shouldn't be between parent and child elements